aws/src/test/java/org/apache/iceberg/aws/glue/TestIcebergToGlueConverter.java

```diff
@@ -384,4 +384,71 @@ public void testSetTableInputInformationWithExistingTable() {
         .as("Columns should match")
         .isEqualTo(expectedTableInput.storageDescriptor().columns());
   }
+
+  @Test
+  public void testSetTableInputInformationWithExistingTableWithoutStorageDescriptor() {
```

Comment:
Can please use [updateTableColumns](https://github.com/apache/iceberg/blob/12737a1282d2e82432c0e5924b3fe33dc4ffb109/aws/src/integration/java/org/apache/iceberg/aws/glue/GlueTestBase.java#L161) for the test?